packages/replay/src/replay.ts

```diff
@@ -198,6 +198,22 @@ export class ReplayContainer implements ReplayContainerInterface {
         // instead, we'll always keep the last 60 seconds of replay before an error happened
         ...(this.recordingMode === 'error' && { checkoutEveryNms: ERROR_CHECKOUT_TIME }),
         emit: this._handleRecordingEmit,
+        onMutation: (mutations: unknown[]) => {
+          const count = mutations.length;
+
+          if (count > 500) {
+            const breadcrumb = createBreadcrumb({
+              category: 'replay.mutations',
+              message: `A mutation with ${count} changes was recorded, which may indicate slow performance.`,
+              data: {
+                mutationsCount: count,
```

Comment:
Let's make this a bit more generic and use `value`?

Reply:
not sure but may be _too_ generic? As what does `value` mean when looking at `replay.mutations` 😅 I could make it just `count` or `length` maybe?